src/tests/rect.c

```diff
@@ -17,6 +18,12 @@ GRAPHENE_TEST_UNIT_BEGIN (rect_init)
   graphene_rect_init_from_rect (&s, r);
   g_assert_true (graphene_rect_equal (&s, r));
 
+  zero = GRAPHENE_RECT_INIT_ZERO;
+  g_assert_cmpfloat (zero.origin.x, ==, 0.f);
```

Comment:
Don't use `g_assert_cmpfloat()` any more: it's a bad macro.

Use `graphene_assert_fuzzy_rect_equal (&zero, graphene_rect_zero (), 0.0001f)` instead.

Reply:
Done